sh/tests/calculator_test.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+# set -euo pipefail
+IFS='
+'
+
+script_dirS=$(cd -P "$(dirname "$BASH_SOURCE")" &>/dev/null && pwd)
+
+challenge() {
+    submitted="./calculator.sh $@
+    "
+    expected="./calculator.sh $@
+    "
+    submitted+=$(2>&1 bash "$script_dirS"/student/calculator.sh "$@")
+    submitted+="
+    exit status: $?"
+    expected+=$(2>&1 bash "$script_dirS"/solutions/calculator.sh "$@")
+    expected+="
+    exit status: $?"
+
+    diff -U 1 <(echo "$submitted") <(echo "$expected")
+    if [ $? != 0 ]
+    then
+        exit 1
+    fi
+}
+
+# Check if student uses case statement
+if [[ $(cat "$script_dirS"/solutions/calculator.sh | grep case | wc -l) -eq 0 ]]
```

Comment:
here you are checking in the solution but it must be the student.
"cat "$script_dirS"/student/calculator.sh "

Reply:
done